<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_CamTitleBarWidget</class>
 <widget class="QWidget" name="C_CamTitleBarWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>40</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>40</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>40</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>C_CamTitleBarWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,0,0,0,0,0">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="stw::opensyde_gui_elements::C_CamOgePubProjOp" name="pc_PushButtonSave">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="stw::opensyde_gui_elements::C_OgeFraSeparator" name="pc_Frame">
     <property name="minimumSize">
      <size>
       <width>1</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="stw::opensyde_gui_elements::C_CamOgePubProjOp" name="pc_PushButtonNew">
     <property name="text">
      <string>New</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="stw::opensyde_gui_elements::C_CamOgeTobRecentProjects" name="pc_ToolButtonLoad">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="stw::opensyde_gui_elements::C_CamOgePubProjOp" name="pc_PushButtonSaveAs">
     <property name="text">
      <string>Save as</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="stw::opensyde_gui_elements::C_CamOgePubAbout" name="pc_PushButtonAbout">
     <property name="text">
      <string>About</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="stw::opensyde_gui_elements::C_CamOgePubProjOp" name="pc_PushButtonHelp">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>100</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="pc_LogoLabel">
     <property name="minimumSize">
      <size>
       <width>95</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>95</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_OgeFraSeparator</class>
   <extends>QFrame</extends>
   <header>C_OgeFraSeparator.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_CamOgePubAbout</class>
   <extends>QPushButton</extends>
   <header>C_CamOgePubAbout.hpp</header>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_CamOgePubProjOp</class>
   <extends>QPushButton</extends>
   <header>C_CamOgePubProjOp.hpp</header>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_CamOgeTobRecentProjects</class>
   <extends>QToolButton</extends>
   <header>C_CamOgeTobRecentProjects.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
